index.js

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line no-unused-vars 
+function validatePassword(password) { 
+
+  function length(password) {
+    if(password.length < 8) {
+      return false
+    }
+    else {
+      return true 
+    }
+  }
+  function upperCase(password){ 
+    if (password.toUpperCase() != password) {
+  
+      return true
+    }
+    else {
+      return false
+    }
+  }
+  function LowerCase(password){
+    if (password.toLowerCase() != password) 
+    {
+      return true
+    }
+    else
+    {
+      return false
+    }
+  }
+
+  function character (password){
+    if (password.includes('!', '#', '$', '@')){
+      return true
+    }
+    else {
+      return false 
+    }
+    
+  }
+
+  function integer (password){
+    if (password.includes('0', '1', '2', '3', '4', '5', '6', '7', '8', '9')){
+      return true
+    }
+    else {
+      return false
+    }
+  }
```

Comment:
1) Predicate functions should use `is` or `has` in the name so `hasNumeric`.
2) That's not how `includes` works. This function will always return false. See: [MDN String includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes)
3) You don't need an if when you are return true/false. You just return the condition itself.